site/public/js/ta-grading.js

```diff
@@ -805,10 +847,10 @@ function setPanelsVisibilities (ele, forceVisible=null, position=null) {
   panelElements.forEach((panel) => {
     let id_str = document.getElementById(panel.str + "_btn") ? "#" + panel.str + "_btn" : "#" + panel.str + "-btn";
     if (panel.str === ele) {
-      const eleVisibility = forceVisible !== null ? forceVisible : !$("#" + panel.str).is(":visible");
+      let eleVisibility = forceVisible !== null ? forceVisible : !$("#" + panel.str).is(":visible");
```

Comment:
Any particular reason for this change?  I may have missed it but I don't see this variable being modified anywhere.